Linphone/Presentation.Dto/CallViewModel.cs

```diff
@@ -47,7 +47,7 @@ public class CallViewModel
     public enum CallState
```

Comment:
Please use sip standard call state, and write description comments on each state into code with xml style

Reply:
It's used on our call center domain and customized to the customer needs.
we do not need this here and i will remove it from CallViewModel.

Reply:
So please remove it

Reply:
My fault. we have 2 call state enum first is used on Call Flow Control functionality, second one is used in our panel domain. 

Third is Linphone default call state. I removed "Panel Domain one" but for "Call Flow Control" i had to choose a better name as long as it's not necessarily call state. so i renamed that to Phone State (For State transition purposes).